.github/workflows/conformance-k8s-network-policies.yaml

```diff
@@ -93,7 +93,7 @@ jobs:
              --wait \
              --namespace kube-system \
              --set nodeinit.enabled=true \
-             --set kubeProxyReplacement=partial \
+             --set kubeProxyReplacement=true \
              --set socketLB.enabled=false \
              --set externalIPs.enabled=true \
              --set nodePort.enabled=true \
```

Comment:
I think you want `=false` here, and let all the specific feature options take effect.

Reply:
ah good spot :bow: